packages/vsce/src/commands/setFocusRegionCommand.ts

```diff
@@ -28,7 +28,7 @@ import { CICSLogger } from "../utils/CICSLogger";
 
 export function setFocusRegionCommand() {
   return commands.registerCommand("cics-extension-for-zowe.setFocusRegion", async () => {
-    await updateFocusRegion();
+    await getFocusRegion();
```

Comment:
I'm not sure if this was intentional or if this was a result of manual testing. The `Set` command suggests it will update it, so `updateFocusRegion` makes sense in my mind.